integrated_channels/sap_success_factors/transmitters/courses.py

```diff
@@ -18,16 +18,18 @@ class SuccessFactorsCourseTransmitter(SuccessFactorsTransmitterBase):
     This endpoint is intended to carry out an export of course data to SuccessFactors for a given Enterprise.
     """
 
-    def transmit(self, payload):
+    def transmit_block(self, serialized_payload):
         """
-        Send a course data import call to SAP SuccessFactors using the client.
+        SAPSuccessFactors can only send 1000 items at a time, so this method sends one "page" at a time.
 
         Args:
-            payload (SapCourseExporter): The OCN course exporter object to send to SAP SuccessFactors
+            serialized_payload (bytes): A set of bytes containing a page's worth of data
+
+        Returns:
+            code: An integer status for the HTTP request
+            body: The server's response body
```

Comment:
Also please clarify who the "server" is in this case -- eg, the "SAP SuccessFactors server"